templates/aws-standalone-cp/templates/awscluster.yaml

```diff
@@ -2,6 +2,8 @@ apiVersion: infrastructure.cluster.x-k8s.io/v1beta2
 kind: AWSCluster
 metadata:
   name: {{ include "cluster.name" . }}
+  finalizers:
```

Comment:
It makes no sense to add it for standalone. Remove.

Reply:
I've retained this finalizer: I encountered several times when Machines were left behind after AWSCluster is removed.

Reply:
I just encountered this behavior in test, the `AWSMachine` objects still exist even with the `AWSCluster` gone:

```
k get awsmachines.infrastructure.cluster.x-k8s.io  -n hmc-system
NAME                              CLUSTER            STATE     READY   INSTANCEID                              MACHINE
squizzi-e2e-test-cp-0             squizzi-e2e-test   running   true    aws:///us-west-1a/i-003dbb531c5686cad   squizzi-e2e-test-cp-0
squizzi-e2e-test-md-xjskf-bg9hf   squizzi-e2e-test   running   true    aws:///us-west-1a/i-0c12931c54ef9d1db   squizzi-e2e-test-md-xjskf-bg9hf
```

```
k get -n hmc-system awscluster
No resources found in hmc-system namespace.
```